### PR TITLE
Add script to update kubelet

### PR DESCRIPTION
**Pull request by indradhanush**

### CODE REVIEW

File: scripts/updates/update-kubelet [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+wait_for_node_ready() {
+  local state
+  local type
+  local status
+  local counter="0"
+
+  while [[ $counter -lt 15 ]]; do
+    state=$(kubectl get nodes "${node}" -o json | jq -r ".status.conditions[-1]")
+    status=$(echo "${state}" | jq -r ".status")
```
Comment (surajssd) [inline scripts/updates/update-kubelet]: Same thing can be achieved by running

```
kubectl get nodes minikube -o jsonpath='{.status.conditions[-1:].status}'
```

File: scripts/updates/update-kubelet [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    type=$(echo "${state}" | jq -r ".type")
```
Comment (surajssd) [inline scripts/updates/update-kubelet]: similarly this:

```
kubectl get nodes minikube -o jsonpath='{.status.conditions[-1:].type}'
```

More ref: https://kubernetes.io/docs/reference/kubectl/jsonpath/

File: scripts/updates/update-kubelet (hunk 1 above)
Comment (indradhanush) [reply scripts/updates/update-kubelet]: Using negative indexing raised a panic for me.

File: scripts/updates/update-kubelet (hunk 2 above)
Comment (indradhanush) [reply scripts/updates/update-kubelet]: Not sure how to retrieve two separate keys in the same `jsonpath` filter. Separating by `comma` didn't work. I can switch to `jsonpath` on line 31 though but I'll still have to stick to `jq` on lines 32 and 33.

Comment (indradhanush) [reply scripts/updates/update-kubelet]: Nevermind. Looked up the docs. It's doable. :)

**Comment by indradhanush** [comment]
Merged on the commandline.

**Pull request merged**